### DISCUSSION TITLE
Welcome to aframe-vr-player Discussions!

### DISCUSSION THREAD

**Discussion by mysterion** [mysterion/aframe-vr-player#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ConnorMusk** [comment]
Hi! Love to see people bringing likeminded people together in the VR space!

My name is Cro, I am a VR fanatic that loves everything VR and sees a big future for it. I eagerly await the day I can live a Ready Player One like lifestyle so I can experience an infinite of experiences. I have recently setup a VR blog which I like to cover the current news and advancements, hardware and games, well, anything and everything VR. I am more on the interest side than the technical ability side, but I am keen to learn and start some of my own VR projects. In addition to this, as like you, I am keen to help spread peoples VR projects to the wider community, so if anyone is working on any sort of VR project and wants to share it with the wider community, give me a message and we can make you a feature piece on our blog thevrfuture.com


Looking forward to some interesting discussions!

- Cro
